docs/local_install_skaffold.md

```diff
@@ -1,62 +1,293 @@
+# Local installation of Substra using Kubernetes and Skaffold
+
+> This is an ongoing document, please feel free to reach us or to raise any issue.
+
+This guide will help you to run the Substra platform on your machine in development mode, with a two nodes setup.
+
+- [Local installation of Substra using Kubernetes and Skaffold](#local-installation-of-substra-using-kubernetes-and-skaffold)
+  - [Substra Setup](#substra-setup)
+    - [General knowledge](#general-knowledge)
+    - [Hardware requirements](#hardware-requirements)
+    - [Software requirements](#software-requirements)
+      - [Kubernetes](#kubernetes)
+        - [Running Kubernetes locally](#running-kubernetes-locally)
+        - [Installing Kubectl](#installing-kubectl)
+      - [Helm](#helm)
+      - [Skaffold](#skaffold)
+    - [Virtualization](#virtualization)
+    - [Get the source code (Mac & Ubuntu)](#get-the-source-code-mac--ubuntu)
+    - [Configuration](#configuration)
+      - [Minikube (Ubuntu)](#minikube-ubuntu)
+      - [Helm init (Mac & Ubuntu)](#helm-init-mac--ubuntu)
+      - [Network](#network)
+  - [Running the platform](#running-the-platform)
+    - [Start Substra](#start-substra)
+      - [1. hlf-k8s repository](#1-hlf-k8s-repository)
+      - [2. substra-backend repository](#2-substra-backend-repository)
+      - [3. substra-frontend repository](#3-substra-frontend-repository)
+    - [Stop Substra](#stop-substra)
+    - [Reset Substra](#reset-substra)
+  - [Login, password and urls](#login-password-and-urls)
+    - [Credentials and urls](#credentials-and-urls)
+    - [Browser extension](#browser-extension)
+    - [Substra CLI config & login (Mac & Ubuntu)](#substra-cli-config--login-mac--ubuntu)
+  - [Troubleshooting](#troubleshooting)
+    - [Virtualization issues](#virtualization-issues)
+    - [Kubectl useful commands](#kubectl-useful-commands)
+    - [Minikube useful commands](#minikube-useful-commands)
+    - [Tiller](#tiller)
+    - [[WIP] Ongoing issues](#wip-ongoing-issues)
+  - [Need help?](#need-help)
+  - [Further resources](#further-resources)
+  - [Acknowledgements](#acknowledgements)
+
+___
+
+When everything is ready, you will be able to start Substra with:
 
-This guide will help you run the Substra platform on your machine in development mode, with a 2 organisations setup.
+```sh
+# If you use Minikube
+minikube start --cpus 6 --memory 8192 --disk-size 50g --kubernetes-version='v1.15.4'
+
+# In 3 different terminal windows, in this order:
+# In the repository hlf-k8s
+skaffold dev
+
+# In the repository susbtra-backend
+skaffold dev
+
+# In the repository substra-frontend
+skaffold dev
+```
+
+## Substra Setup
+
+### General knowledge
+
+In order to install Substra, it is *recommended* to be comfortable with your package manager and to have some basic knowledge of Unix (Mac or Ubuntu/Debian) administration and network. It might also be useful to have a good command of Docker containers and Kubernetes orchestration.
```

Comment:
```suggestion
In order to install Substra, it is *recommended* to be comfortable with your package manager and to have some basic knowledge of Unix (Mac or Ubuntu/Debian) administration and network. It might also be useful to have a good knowledge of Docker containers and Kubernetes orchestration.
```